distribution/src/main/resources/bin/elasticsearch-keystore

```diff
@@ -77,10 +77,9 @@ HOSTNAME=`hostname | cut -d. -f1`
 export HOSTNAME
 
 declare -a args=("$@")
-path_props=(-Des.path.home="$ES_HOME")
```

Comment:
this is super nit picky, but I wonder if we should leave this or change the other cli tools like elasticsearch-plugin to be consistent in how we handle this?